starport/services/networkbuilder/config.go

```diff
@@ -3,20 +3,20 @@ package networkbuilder
 import (
 	"encoding/json"
 	"os"
-	"path/filepath"
 
+	"github.com/tendermint/starport/starport/pkg/xfilepath"
 	"github.com/tendermint/starport/starport/services"
 )
 
 var (
-	confPath = filepath.Join(services.StarportConfDir, "networkbuilder")
-)
+	confDir = "networkbuilder"
 
-func init() {
-	if err := os.MkdirAll(services.StarportConfDir, 0755); err != nil {
-		panic(err)
-	}
-}
+	// confPath returns the path of Starport Network configuration
+	confPath = xfilepath.Join(
+		services.StarportConfPath,
+		xfilepath.Path(confDir),
```

Comment:
value of `confDir` can be added here and `confDir` can be removed if it isn't used anywhere else.